src/asdf-utils.lisp

```diff
@@ -0,0 +1,82 @@
+(in-package #:sbcl-librarian)
+
```

Comment:
If possible this file should define a custom asdf operation instead of monkey patching an existing one.

Reply:
added a TODO